frontend/src/metabase/visualizations/components/ChartSettings/ChartSettings.tsx

```diff
@@ -356,60 +356,138 @@ export const ChartSettings = ({
           />
         </ChartSettingsListContainer>
       </ChartSettingsMenu>
-      {!noPreview && (
-        <ChartSettingsPreview>
-          <SectionWarnings warnings={warnings} size={20} />
-          <ChartSettingsVisualizationContainer>
-            <Visualization
-              className={CS.spread}
-              rawSeries={chartSettingsRawSeries}
-              showTitle
-              isEditing
-              isDashboard
-              dashboard={dashboard}
-              dashcard={dashcard}
-              isSettings
-              showWarnings
-              onUpdateVisualizationSettings={handleChangeSettings}
-              onUpdateWarnings={setWarnings}
-            />
-          </ChartSettingsVisualizationContainer>
-          <ChartSettingsFooter
-            onDone={handleDone}
-            onCancel={handleCancel}
-            onReset={onResetToDefault}
-          />
-        </ChartSettingsPreview>
-      )}
       <ChartSettingsWidgetPopover
         anchor={popoverRef as HTMLElement}
         widgets={[styleWidget, formattingWidget].filter(
           (widget): widget is Widget => !!widget,
         )}
         handleEndShowWidget={handleEndShowWidget}
       />
+    </>
+  );
+};
+
+export const QuestionChartSettings = ({
+  question,
+  widgets: propWidgets,
```

Comment:
Can you double-check this? I don't think we pass `widgets` anywhere.

```suggestion
```

If so, we could probably simplify the types a bit.

Reply:
We don't use `widgets` anywhere except for tests 🫠 I thought about removing the prop and redoing the tests, but as of now we don't have a way to test these components otherwise without doing a massive refactor of the tests. We would have to create a bunch of mock elements and things to make things work. In an ideal world I would love to do this but I think we really should do this in another PR

Reply:
The only reason I think it's ok for now is because we already do this on `master` and we have a ton of E2E + unit tests that use the visualization settings already

Reply:
OK that's fair, I'd love to at least see that pointed out in the type e.g. you could name a type like
```
type TestType = {
  widgets?: Widget[];
```

Feel free to come up with a better name, but make it obvious that it's for test. At least this is deliberate and doesn't confuse anyone. What do you think?

Reply:
oh that's a great idea. let me do that